src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -68,11 +68,11 @@
         BUILD = Core.class.getPackage().getImplementationVersion();
     }
 
-    private Core() {
-        //Private Constructor
-    }
-
-    // TODO: Doc
+    /**
+     * Boots the Titan framework in its entirety, initialising all static fields.
+     * @throws IOException
+     * @throws SQLException
+     */
     public static void start() throws IOException, SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.sql.SQLException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1160)